web/src/pages/Environments/NoResults.tsx

```diff
@@ -0,0 +1,15 @@
+import {FC} from 'react';
+import {Typography} from 'antd';
+import * as S from './Envs.styled';
+
+const NoResults: FC = () => {
+  return (
```

Comment:
Is this any different from what we use for tests?

Reply:
i wasnt being used. Just carried over 
Just removed ✅